src/ResourceManagement/Compute/Domain/InterfaceImpl/VirtualMachineImpl.cs

```diff
@@ -1896,16 +1896,6 @@ VirtualMachine.Update.IUpdate VirtualMachine.Update.IUpdate.WithSize(string size
             return this.WithSize(sizeName);
         }
 
-        /// <summary>
-        /// Specifies a new size for the virtual machine.
-        /// </summary>
-        /// <param name="size">A size from the list of available sizes for the virtual machine.</param>
-        /// <return>The next stage of the definition.</return>
-        VirtualMachine.Update.IUpdate VirtualMachine.Update.IUpdate.WithSize(VirtualMachineSizeTypes size)
-        {
-            return this.WithSize(size);
-        }
-
```

Comment:
This is what I was saying about the side effect of removing `VirtualMachineSizeTypes` :(